### ISSUE TITLE
View posts by user should (optionally) include resteems.

### ISSUE THREAD

**Issue by antonchanning** [blockpress/blockpress.me#46]
At the moment a users posts, as viewed on either the post listing page, or their profile page, excludes resteems. It would be good if it could optionally include them.

**Comment by antonchanning** [comment]
https://jsfiddle.net/bonustrack/24b7ky2s/2/

^ This should help.

**Comment by DanaVarahi** [comment]
In my opinion option, 2 is better. It seems more logical that this change/decision should be determined in the config file and not in the theme.

**Comment by antonchanning** [comment]
I agree. If a site admin wants one or the other behaviour, they won't want changing a theme to mess it up. Option 2 gives them more flexibility.
